### PR TITLE
Alex Update for HW3

### PR DESCRIPTION
**Pull request by clairmont**
Starting to get the daily balance and HTML generation ideas from your example; will be incorporating those methods as I continue to grasp it!

### CODE REVIEW

File: assignments/assignment03.rb [hunk 1]
```diff
@@ -23,6 +34,27 @@
 
 # re-implement mean, median, to_sentence as methods on Array
 
+class Array
```
Comment (planetcohen) [inline assignments/assignment03.rb:37]: nicely done

File: assignments/assignment03.rb [hunk 2]
```diff
@@ -7,6 +7,17 @@
 
 # re-implement titleize and palindrome? as methods on String
 
+class String
```
Comment (planetcohen) [inline assignments/assignment03.rb:10]: nicely done

File: assignments/assignment03.rb [hunk 3]
```diff
@@ -41,6 +73,120 @@
...
+ require 'csv'
+ require 'date'
+
+	def import
+		@trans = CSV.read('C:\Users\01\Desktop\assignment02-input.csv', headers:true, header_converters: :symbol, converters: :all).collect { |row| Hash[row.collect {|col,row| [col,row] }] }
+
+	end
+
+	def starting_balance
+		"0"
```
Comment (planetcohen) [inline assignments/assignment03.rb:86]: curious that you are returning a string instead of a number

**Comment by planetcohen** [comment]
looks like the BankAccount problem isn't done yet

**Comment by clairmont** [comment]
I'm going to keep working on that tonight, I wasn't seeing how to get my
rendering working but I've been reviewing your solution and I think I'm
getting it; working on it more this evening in addition to this weeks
assignment!!

**Pull request closed without merging**